scripts/reexport-omnivoice/exporters.py

```
@@ -0,0 +1,173 @@
+import torch, omnivoice  # noqa: F401
+from transformers import AutoModel
+
+
+def load_model(model_dir):
+    return AutoModel.from_pretrained(model_dir, trust_remote_code=True, dtype=torch.float32,
+                                     attn_implementation="eager").eval()
+
+
+class BidiLLM(torch.nn.Module):
+    def __init__(self, llm):
+        super().__init__(); self.llm = llm
```

Ruff (0.15.21)

[error] Multiple statements on one line (semicolon)

(E702)